sensor_array/analysis/execute_brute_force_analysis.py

```diff
@@ -13,16 +14,22 @@
     import_experimental_data,
     import_simulated_data,
     calculate_element_pmf,
+    calculate_array_pmf,
+    calculate_all_arrays_list,
     calculate_all_arrays,
+    calculate_single_array_kld,
     create_bins,
+    create_comp_set_dict,
     bin_compositions,
+    bin_compositions_single_array,
     calculate_kld,
     choose_arrays,
     assign_array_ids,
     save_element_pmf_data,
     save_unbinned_array_pmf_data,
     save_binned_array_pmf_data,
     plot_binned_array_pmf_data)
```

Comment:
this many imports is a sign you either should be using a namespace, i.e. calling these as brute_force_analysis.choose_arrays, or maybe the method that is calling all these should also be in the brute_force_analysis namespace